int32 plane_id         # the id for the scanning
float32 height_std
float32 height_mean
float32 heighest_max_min_difference
float32 tilt_angle
